From 14f6da1874bf6e516584e437e272472f03853a09 Mon Sep 17 00:00:00 2001
From: Xianzhong <b07117@freescale.com>
Date: Tue, 26 Feb 2013 18:38:59 +0800
Subject: ENGR00243077 Fixed JB4.2 multi-user switch hang issue

Gpu virtual memory has potiential hang issue exposed,
Vivante will fix this hang issue with 4K contigueous memory limit.

4K limit is needless now and should be disabled by default.

Signed-off-by: Xianzhong <b07117@freescale.com>
Acked-by: Lily Zhang
---
 .../mxc/gpu-viv/hal/kernel/inc/gc_hal_options.h    |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/mxc/gpu-viv/hal/kernel/inc/gc_hal_options.h b/drivers/mxc/gpu-viv/hal/kernel/inc/gc_hal_options.h
index 05c9fcb..86e9133 100644
--- a/drivers/mxc/gpu-viv/hal/kernel/inc/gc_hal_options.h
+++ b/drivers/mxc/gpu-viv/hal/kernel/inc/gc_hal_options.h
@@ -785,7 +785,7 @@
         limited by gcdCONTIGUOUS_SIZE_LIMIT.
  */
 #ifndef gcdCONTIGUOUS_SIZE_LIMIT
-#   define gcdCONTIGUOUS_SIZE_LIMIT             4096
+#   define gcdCONTIGUOUS_SIZE_LIMIT             0
 #endif
 
 #ifndef gcdDISALBE_EARLY_EARLY_Z
-- 
1.7.9.5

